subsys/lorawan/radio/sx1272.c

```diff
@@ -0,0 +1,187 @@
+#include <gpio.h>
+#include <spi.h>
+
+#include <lorawan/sx1276.h>
+
+extern DioIrqHandler *DioIrq[];
+
+static struct spi_cs_control spi_cs = {
+	.gpio_pin = 3,
+	.delay = 0,
+};
+
+static spi_config spi_config = {
+	.frequency = 16000000,
+	.operation = SPI_OP_MODE_MASTER | SPI_MODE_CPOL |
+		SPI_MODE_CPHA | SPI_WORD_SET(8) | SPI_LINES_SINGLE,
+	.slave = 1,
+	.cs = &spi_cs,
+};
+
+static char dio_gpio_dev_name_table[6] = {
+	CONFIG_SX1272_DIO0_DEV_NAME,
+	CONFIG_SX1272_DIO1_DEV_NAME,
+	CONFIG_SX1272_DIO2_DEV_NAME,
+	CONFIG_SX1272_DIO3_DEV_NAME,
+	CONFIG_SX1272_DIO4_DEV_NAME,
+	CONFIG_SX1272_DIO5_DEV_NAME,
+};
+
+static uint8_t dio_gpio_pin_table[6] = {
+	CONFIG_SX1272_DIO0_PIN,
+	CONFIG_SX1272_DIO1_PIN,
+	CONFIG_SX1272_DIO2_PIN,
+	CONFIG_SX1272_DIO3_PIN,
+	CONFIG_SX1272_DIO4_PIN,
+	CONFIG_SX1272_DIO5_PIN,
+}
+
+int bus_spi_init(void)
+{
+	spi_cs.gpio_dev = device_get_binding("GPIOA");

```

Comment:
Ok, so here you will have to use your Kconfig options you declare in the last patch